### PR TITLE
Redisdead v1.8

### PR DESCRIPTION
**Pull request by regit**
Update of #1521:
- Fix coccinelle check
- Add sensor-name in yaml

I'm almost ok with @jasonish suggestion on updating the other eve event. But we need to discuss the details a bit so maybe a later PR.

PR builds:
- PR regit: https://buildbot.openinfosecfoundation.org/builders/regit/builds/79
- PR regit-pcap: https://buildbot.openinfosecfoundation.org/builders/regit-pcap/builds/77

### CODE REVIEW

File: src/util-logopenfile.c [hunk 1]
```diff
@@ -311,3 +463,108 @@ int LogFileFreeCtx(LogFileCtx *lf_ctx)
...
     SCReturnInt(1);
 }
+
+#ifdef HAVE_LIBHIREDIS
+static int  LogFileWriteRedis(LogFileCtx *file_ctx, char *string, size_t string_len)
+{
+    if (file_ctx->redis == NULL) {
+        SCConfLogReopenRedis(file_ctx);
+        if (file_ctx->redis == NULL) {
+            SCMutexUnlock(&file_ctx->fp_mutex);
```
Comment (inliniac) [inline src/util-logopenfile.c]: where is the lock?

Comment (regit) [inline src/util-logopenfile.c]: Rhaaa, I should not code at 1am after a big day of work.

File: src/util-logopenfile.c [hunk 2]
```diff
@@ -311,3 +463,108 @@ int LogFileFreeCtx(LogFileCtx *lf_ctx)
...
+            SCLogInfo("Reconnected to redis server");
+        }
+    }
+    /* TODO go async here ? */
+    if (file_ctx->redis_setup.batch_size) {
+        redisAppendCommand(file_ctx->redis, "%s %s %s",
+                file_ctx->redis_setup.command,
+                file_ctx->redis_setup.key,
+                string);
+        if (SC_ATOMIC_CAS(&file_ctx->redis_setup.batch_count, file_ctx->redis_setup.batch_size, 0)) {
```
Comment (inliniac) [inline src/util-logopenfile.c]: I don't understand this CAS call. Actually, I think we have the file_ctx lock here right? So why do we need atomics?

**Comment by inliniac** [comment]
Can "util-logopenfile: suppress unused batch timeout" be squashed into the commit that adds the code?

File: src/util-logopenfile.c (hunk 1 above)
Comment (regit) [inline src/util-logopenfile.c]: OK, fixed.

File: src/util-logopenfile.c (hunk 2 above)
Comment (regit) [inline src/util-logopenfile.c]: Yes, my initial code was without that so I needed a lock but as hiredis is not thread safe I added back the lock. So yes, it is mainly useless now. I will try without it.

**Comment by regit** [comment]
Humm, yes :) easier than thought and done.

**Comment by inliniac** [comment]
On 06/01/2015 10:59 PM, Eric Leblond wrote:
> Humm, yes :) easier than thought and done.

Often in such cases it's easiest to edit the original commit instead of
squashing another into it.

**Comment by regit** [comment]
I was lucky on this one: direct rebase was working :)

**Pull request closed without merging**